src/style.css

```diff
@@ -1,3 +1,64 @@
-body {
+* {
   margin: 0;
 }
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
- Don't use * selector (it impacts performance)
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/2fb86c69-51bd-43b0-bdce-dbce03b80d9f)

Reply:
tried to add it to other elements but then problems with the tests
can you recommend how to fix this?